lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group.py

```diff
@@ -52,6 +52,13 @@
         - "Valid values are: [1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 3653]"
       required: false
       type: int
+    purge_retention_policy:
+      description:
+        - "Wether to purge the retention policy or not. Take precedence over retention when used together."
```

Comment:
```suggestion
        - "Whether to purge the retention policy or not. Takes precedence over retention when used together."
```

I'm not a big fan of "Takes precedence over".  IMO it would be better to user "mutually_exclusive" in the AnsibleModule definition (this is tested before the defaults are set)